tests/android/tests/test_android_settings.py

```diff
@@ -0,0 +1,141 @@
+
+
+class TestAndroidSettings:
+    """
+    Settings screen's Test Case
+    """
+
+    def test_start_settings_screen_smoke(self, login, set_capabilities, setup_logging):
+        """
+        Scenarios:
+            Verify Main Dashboard screen is loaded successfully
+        """
+
+        setup_logging.info(f'Starting {TestAndroidSettings.__name__} Test Case')
+        global_contents = Globals(setup_logging)
+        whats_new_page = AndroidWhatsNew(set_capabilities, setup_logging)
+
+        if login and global_contents.whats_new_enable:
+            assert whats_new_page.navigate_features().text == 'Done'
+            whats_new_page.get_done_button().click()
+
+    def test_validate_ui_elements(self, set_capabilities, setup_logging):
+        """
+        Scenarios:
+            Verify that Profile screen will show following contents:
+                Back icon, "Profile" as Title, Edit
+                Profile Image, User Name, Video Settings
+                Support Info, Contact Support, Terms of Use
+                Privacy Policy, Cookie Policy, Personal Info
+                View FAQ, App Version, Up to Date
+                Logout button
+        """
+
+        main_dashboard_page = AndroidMainDashboard(set_capabilities, setup_logging)
+        profile_page = AndroidProfile(set_capabilities, setup_logging)
+        global_contents = Globals(setup_logging)
+
+        profile_tab = main_dashboard_page.get_profile_tab()
+        assert profile_tab.get_attribute('content-desc') == values.MAIN_DASHBOARD_PROFILE_TAB
+        assert profile_tab.get_attribute('selected') == values.FALSE_LOWERCASE
+        profile_tab.click()
+        profile_page.get_settings_button().click()
+
+        assert profile_page.get_settings_screen_title().text == values.PROFILE_SETTINGS_UPPER_TEXT
+        manage_account_label = profile_page.get_manage_account_label()
+        assert manage_account_label.text == values.PROFILE_MANAGE_ACCOUNT_LABEL
+
+        video_label = profile_page.get_video_label()
+        assert video_label.text == values.PROFILE_VIDEO_LABEL
+        assert profile_page.get_profile_txt_support_info().text == values.PROFILE_SUPPORT
+        assert profile_page.get_profile_txt_contact_support().text.lower() == values.PROFILE_CONTACT_SUPPORT
+        assert profile_page.get_profile_txt_terms_of_use().text == values.PROFILE_TERMS_OF_USE_UPPERCASE
+        assert profile_page.get_profile_txt_privacy_policy().text.lower() == values.PROFILE_PRIVACY_POLICY
+
+        global_contents.scroll_from_element(set_capabilities, profile_page.get_profile_txt_privacy_policy())
+        assert profile_page.get_profile_txt_cookie_policy().text.lower() == values.PROFILE_COOKIE_POLICY
+        assert profile_page.get_profile_personal_info().text == values.PROFILE_PERSONAL_INFO
+        assert profile_page.get_profile_txt_view_faq().text == values.PROFILE_FAQ
+        # assert profile_page.get_profile_app_version_code().text == values.ANDROID_APP_VERSION
```

Comment:
do we need this line?

Reply:
good catch, it was commented mistakenly. 
fixed now